PyInstaller/hooks/hook-faker.py

```diff
@@ -0,0 +1,3 @@
+from PyInstaller.utils.hooks import collect_submodules, collect_data_files
+hiddenimports = collect_submodules('faker')
```

Comment:
Why is there a need to include all submodules? Please show the code in the source. Thanks